### ISSUE TITLE
Make connection_count route also work for meeting admins

### ISSUE THREAD

**Issue by luisa-beerboom** [OpenSlides/openslides-autoupdate-service#749]
Meeting admins should be allowed to call the connection_count route.
The return value for them should only include the users in the meetings where they are admin

**Comment by ostcar** [comment]
The route returns a dictionary from user id to connection amount.

It is currently not possible to now, to which meeting a connection belongs.

What should I do with the amount, that could belong to a different meeting. I could just return it anyway or (for a meeting admin) change the value for every user to 1. So a meeting admin only gets the user IDs but not the amount.

What should I do with the anonymous user. Should it be included in the response of a meeting admin?

**Comment by luisa-beerboom** [comment]
No, it is quite clear that discerning between connections in that way would be a lot of work if not impossible.
It certainly would be nice, but is not necessary.

When it comes to the connection count, it would be best if the meeting admin can access it, but it's fine as long as the general gist comes across, so they don't need to be accurate to the meeting.
It's completely okay if all he get's is is "X users that are part of your meetings are currently logged in, they have Y connections open".
This loss of accuracy as opposed to the old way of doing things with ICC is a trade-off that we knew we'd have to make.

What we basically want for non-orga-managers is: "The regular response but minus the data of any user who is not part of any meeting, where the requester is in the admin group"

So when there's two meetings A and be with users 1 and 2 being in meeting A and users 1 and 3 being in meeting B and the response for an orga-manager would look like this:
```
{
   1: 2
   2: 4
   3: 3
}
```
it should turn into this
```
{
   1: 2
   3: 3
}
```
for an admin of meeting B who doesn't have any orga-level permissions 

As for the anonymous user: Since the anonymous user feature doesn't even work right now, I don't know.
I'm erring on the side of "If the meeting can be viewed by anonymous users, send the anonymous user data. If not, don't"

**Comment by ostcar** [comment]
Ok. I can do it this way as soon, as I am back in two weeks
